cli/cmds/utils

```diff
@@ -0,0 +1,589 @@
+    echo "==================== BASE URI ===================="
+    echo "Contract Address:             $(gum style --foreground 212 "$(format_address "$contract_address")")"
+    echo "Chain ID:                     $(gum style --foreground 212 "$chain_id")"
+    echo "Base URI:                     $(gum style --foreground 212 "$base_uri")"
+    echo "==================================================="
+    echo ""
+    
+    if ! gum confirm "Do you want to proceed?"; then
+        echo "Exiting..."
+        exit 1
+    fi
+}
+
+confirm_set_global_wallet_limit() {
+    echo ""
+    echo "==================== GLOBAL WALLET LIMIT ===================="
+    echo "Contract Address:             $(gum style --foreground 212 "$(format_address "$contract_address")")"
+    echo "Chain ID:                     $(gum style --foreground 212 "$chain_id")"
+    echo "Global Wallet Limit:          $(gum style --foreground 212 "$global_wallet_limit")"
+    echo "==========================================================="
+    echo ""
+
+    if ! gum confirm "Do you want to proceed?"; then
+        echo "Exiting..."
+        exit 1
+    fi
+}
+
+confirm_set_token_uri_suffix() {
+    echo ""
+    echo "==================== TOKEN URI SUFFIX ===================="
+    echo "Contract Address:             $(gum style --foreground 212 "$(format_address "$contract_address")")"
+    echo "Chain ID:                     $(gum style --foreground 212 "$chain_id")"
+    echo "Token URI Suffix:             $(gum style --foreground 212 "$token_uri_suffix")"
+    echo "==========================================================="
+    echo ""
+    
+    if ! gum confirm "Do you want to proceed?"; then
+        echo "Exiting..."
+        exit 1
+    fi
+}
+
+show_main_title() {
+    gum style \
+	--foreground 212 --border-foreground 212 --border double \
+	--align center --width 40 --margin "1 0" --padding "1" \
+	'Magic Drop CLI' '' 'Create and manage NFT collections'
+}
+
+show_title() {
+    local title="$1"
+    local subtitle="$2"
+    gum style \
+    --foreground 212 --border-foreground 212 --border double \
+    --align center --width 40 --margin "1 0" --padding "1" \
+    "$title" \
+    "$subtitle"
+}
+
+select_chain() {
+    local chain=$(printf "%s\n" "${SUPPORTED_CHAINS[@]}" | cut -d':' -f2 | gum choose)
+    local chain_id=$(printf "%s\n" "${SUPPORTED_CHAINS[@]}" | grep "$chain" | cut -d':' -f1)
+    echo "$chain_id:$chain"
+}
+
+set_rpc_url() {
+    case $1 in
+        1) RPC_URL="https://cloudflare-eth.com" ;; # Ethereum
+        137) RPC_URL="https://polygon-rpc.com" ;; # Polygon
+        8453) RPC_URL="https://mainnet.base.org" ;; # Base
+        42161) RPC_URL="https://arb1.arbitrum.io/rpc" ;; # Arbitrum
+        1329) RPC_URL="https://evm-rpc.sei-apis.com" ;; # Sei
+        33139) RPC_URL="https://curtis.rpc.caldera.xyz/http" ;; # ApeChain
+        *) echo "Unsupported chain id"; exit 1 ;;
+    esac
+
+    export RPC_URL
+}
+
+generate_merkle_root() {
```

Comment:
TBH I hesitate to agree bash script is the best option in this case. This example demonstrates that the complexity of our scripts/cmds is too high for bash to handle elegantly:

- increased complexity - the merkle root logic lives in `generateMerkleRoot.ts` 
- hard to build / debug:
  - `generateMerkleRootPath` is hardcoded. If file is moved, the issue can only be caught in runtime.
  - result need to be filtered using `sed` and regex
- readability is not high:
  - e.g. it's impossible to know the meaning of `false` as the second param to `generateMerkleRootPath`